src/graphql.service.ts

```diff
@@ -5,115 +5,128 @@ import {
+                gql: this.withSubgraphId('5t1nkHHxvmfUvPjwCzDVeP1SMgTAc814HscCXeE9AAeG')
             },
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-mainnet/v1/gn',
+                gql: this.withSubgraphId('R3R96j3nVT8oWKfS9zBzrPkFfMC283jiQoEitQ2SFvy'),
                 local: '',
             },
         },
         // Arbitrum One
         42161: {
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-arbitrum-one/v1/gn',
+                gql: this.withSubgraphId('48MG6smPELHeemANcRvssCWj8rLpBXKisRzRgmW7Pg7o'),
                 local: '',
             },
         },
         // Optimism Mainnet
         10: {
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-optimism/v1/gn',
+                gql: this.withSubgraphId('xUD6iaJk38vmWfEkYm5LwkxeTUADCcEtuujfLCJsHac'),
                 local: '',
             },
         },
         // Boba ETH
         288: {
             [EGraphQLService.AnchorageBridge]: {
-                gql: "https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/anchorage-bridge-boba-eth/v1/gn"
+                gql: this.withSubgraphId("BkDvK8QHM8HRHCQtM6CSBjF4ipH1rW4uLBnMGN28JzwM"),
             },
             [EGraphQLService.DAO]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/dao-boba-eth/v1/gn',
+                gql: this.withSubgraphId('2p7KCVnBeNSnTVwMfJC9NENtEKwZn6hv45CU4Cf6PczW'),
                 local: '',
             },
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-boba-eth/v1/gn',
+                gql: this.withSubgraphId('34UUDYdzZFX7afhysqX7JodzKngJmjsFKJDtWsTxi9UA'),
                 local: '',
             },
         },
         // Boba BNB
         56288: {
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-boba-bnb/v1/gn',
+                gql: 'https://graphql.bnb.boba.network/subgraphs/name/boba-bnb-l2/',
                 local: '',
             },
         },
         // BSC
         56: {
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-bsc/v1/gn',
+                gql: this.withSubgraphId('4Eoi62aX8V1DCebNX4mzPi7rfceFaRAeZ3xt7AYVowpy'),
                 local: '',
             },
         },
         // BNB testnet
         97: {
```

Comment:
can we have explorer for BNB testnet `AnchorageBridge` so we can query the proven transactions and finalized